src/scaffoldmaker/meshtypes/meshtype_3d_bladderurethra1.py

```diff
@@ -822,7 +831,7 @@ def generateUreters(region, nodes, mesh, ureterDefaultOptions,elementsCountAroun
                     elementsCountAroundUreter, trackSurfaceUreter1, ureter1Position, trackSurfaceUreter2,
                     ureter2Position, ureterElementPositionDown, ureterElementPositionAround,
                     xBladder, d1Bladder, d2Bladder, nextNodeIdentifier, nextElementIdentifier,
-                    elementsCountUreterRadial, annulusMeshGroups = []):
+                    elementsCountUreterRadial, urinaryBladderMeshGroup, annulusMeshGroups = []):
```

Comment:
First, perhaps best to rename this function to generateUreterInlets (it previously hadn't mentioned mention Ureters which was why I got Mabelle to change it, but should clarify it's just the region where they enter the bladder).
Main issue I have is that the new argument urinaryBladderMeshGroup group doesn't fit the pattern of annulusMeshGroups.
I suggest you replace both with:
bladderMeshGroups (= bladder, neck) to set on annulus and first row of ostium i.e. ostiumMeshGroups
ureterMeshGroups (= ureter, or None if not defining) to pass as vesselMeshGroups on ostium
Basically I consider the final element on the ostium as it opens into the bladder as part of the bladder, but all the elements in the ostium are in ureter group (as you can make the ostium quite long with many elements which clearly become ureters).
This may not be what you're after, for example you may want them to be only bladder or ureter, not both for the final element. See me to discuss?